analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/SymbolicExecution/CollectorTestCheck.cs

```diff
@@ -78,5 +78,22 @@ public void ValidateTag(string tag, Action<SymbolicValue> action)
 
         public void ValidateExitReachCount(int expected) =>
             exitReachedCount.Should().Be(expected);
+
+        public void ValidateExecutionCompleted() =>
+            executionCompletedCount.Should().Be(1);
+
+        public void ValidatePostProcess(int expected) =>
+            postProcessed.Should().HaveCount(expected);
+
+        public void ValidateOperationsAreNotNull()
+        {
+            foreach (var context in postProcessed)
+            {
+                context.State[context.Operation].Should().NotBeNull();
+            }
+        }
+
+        private SymbolicContext PostProcessedContext(string operation) =>
```

Comment:
_Optional:_ This is currently used only once. I expected it to be used more than once by now. So we can remove the helper and embed it into the invocation side.